apps/src/code-studio/pd/workshop_dashboard/components/workshop_table.jsx

```diff
@@ -265,8 +242,8 @@ export default class WorkshopTable extends React.Component {
     return <SessionTimesList sessions={rowData.sessions} />;
   };
 
-  formatBoolean = bool => {
```

Comment:
The `virtual` field was the only one left that was reading a bool so I adjusted the function name